routing/probability_bimodal.go

```diff
@@ -0,0 +1,238 @@
+package routing
+
+import (
+	"math"
+	"time"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+)
+
+const (
+	// DefaultScaleMsat describes the distribution of funds in the LN. The
+	// value determines how quickly the bimodal distribution drops off from
+	// the edges of a channel. A larger value means, that the drop off is
+	// slow.
+	DefaultScaleMsat = lnwire.MilliSatoshi(300000000)
```

Comment:
Isn't there a way around this constant, for example by making it relative to the channel capacity? Or would that be contrary to empirical findings?

Reply:
I haven't found a good way to get rid of it, making it capacity dependent has the tendency to favor large channels, which is why I thought keeping this value constant would be better, but I can certainly have another look.